src/plugins/core/plugin_consolidate.ts

```diff
@@ -0,0 +1,30 @@
+import { Context } from '../../core/Context';
+import { IBundleWriteResponse } from '../../bundle/Bundle';
+import * as consolidate from 'consolidate';
+
+export function pluginConsolidate(engine: string, options: any) {
+  return async (ctx: Context) => {
+    ctx.ict.awaitOn(
+      'before_webindex_write',
+      async (props: { filePath: string; fileContents: string; bundles: Array<IBundleWriteResponse> }) => {
+        if (typeof consolidate[engine] !== 'function') {
+          ctx.log.error(`The template engine you selected is not available in consolidate: ${engine}`);
```

Comment:
how about ctx.fatal? I would make sense to stop the execution completely.

Reply:
I agree 👍

Reply:
Done